backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5726940_sys_gh18285_Use_Lookup_Instead_Listview.sql

```diff
@@ -0,0 +1,23 @@
+select backup_table('ad_column'); -- backup.ad_column_bkp_20240620_114638_818
+select backup_table('ad_field'); -- backup.ad_field_bkp_20240620_114658_431
+
+UPDATE AD_Column
+SET AD_Reference_ID = 30
+WHERE AD_Reference_ID IN (18, 19)
+;
+
+UPDATE AD_Field
+SET AD_Reference_ID = 30
+WHERE AD_Reference_ID IN (18, 19)
```

Comment:
not sure about this. why are we updating AD_Fields too?
how do we know if the field was set to table/table dir reference by intention or not?

Reply:
Because the reference are overridden in some fields in some windows, if not we will still have fields as list

Reply:
SQL updated to exclude first cases where the column reference = search and field reference != search. Then we know it was set on purpose

Reply:
IMHO, that SQL is not doing that....
i.e. which part of that SQL is checking `column reference = search` ?